cloudformation/anghammarad.template.yaml

```diff
@@ -37,6 +40,11 @@ Resources:
             Action:
             - ses:SendEmail
             Resource: "*"
+      - Statement:
+          - Effect: Allow
+            Action:
+              - s3:GetObject
+            Resource: !Sub arn:aws:s3:::${ConfigBucket}/*
```

Comment:
This will probably be too wide if we switch to using another bucket, but while we have a dedicated bucket this is great.